content/automate/ManageIQ/Infrastructure/VM/Migrate/StateMachines/VMMigrate.class/__class__.yaml

```diff
@@ -178,7 +178,7 @@ object:
       datatype: string
       priority: 9
       owner: 
-      default_value: "/Infrastructure/VM/Migrate/Email/VmMigrateTask_Complete?event=vm_migrated"
+      default_value: "/System/Notification/Email/CloudOrchestrationServiceReconfigureComplete?event=service_reconfigured"
```

Comment:
@billfitzgerald0120 
Is this value correct, isn't it supposed to be for VmMigrateTask instead of CloudOrchestration

Reply:
@mkanoor Yes, I need to fix this